### PR TITLE
Feature/t2 f2 46

### PR DESCRIPTION
**Pull request by Abdulzy**
created a blueprint for the profile page. also added functionality for viewing tuits, likes, and dislikes on the profile page

### CODE REVIEW

**Review by gurishe — COMMENTED** [review]
The profile view actually looks really slick, it's excellent. I'm just wondering about the hardcoded images/info (like the number of tuits) and whether or not that should be kept in?

File: src/components/profile/index.js [hunk 1]
```diff
@@ -27,55 +30,94 @@ const Profile = () => {
...
+        {profile.username}
+        <i className="fa fa-badge-check text-primary"/>
+      </h4>
+      <span className="ps-2">67.6K Tuits</span>
+      <div className="mb-5 position-relative">
+        <img className="w-100" src="../images/nasa-profile-header.jpg"/>
+        <div className="bottom-0 left-0 position-absolute">
+          <div className="position-relative">
+            <img className="position-relative ttr-z-index-1 ttr-top-40px ttr-width-150px"
+                 src="../images/nasa-3.png"/>
```
[the remark this answers is not in the document]
Comment (Abdulzy) [reply src/components/profile/index.js:45]: yes it does cause its just the profile its just a place holder as we are trying to show the likes and dislikes and tuits. also the edit profile and admin button too

**Review by prinjal — APPROVED** [review]
Approved.

File: src/components/profile/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (gurishe) [reply src/components/profile/index.js:45]: Works for me, good to merge

**Review by gurishe — APPROVED** [review]

**Comment by gurishe** [comment]
Oh and just make sure to merge it into sprint-4 branch, not sprint-3

**Comment by gurishe** [comment]
Ha sorry, last thing I just noticed. You probably want to remove the package.lock changes from this commit? Nothing changed in the package.json so I'd assume that was just added in by accident?

**Pull request merged**